doctr/transforms/modules/tensorflow.py

```diff
@@ -77,18 +77,63 @@ def extra_repr(self) -> str:
             _repr += f", preserve_aspect_ratio={self.preserve_aspect_ratio}, symmetric_pad={self.symmetric_pad}"
         return _repr
 
-    def __call__(self, img: tf.Tensor) -> tf.Tensor:
+    def __call__(
+        self,
+        img: tf.Tensor,
+        target: Optional[np.ndarray] = None,
+    ) -> Union[tf.Tensor, Tuple[tf.Tensor, np.ndarray]]:
+
         input_dtype = img.dtype
-        img = tf.image.resize(img, self.output_size, self.method, self.preserve_aspect_ratio)
+        if isinstance(self.output_size, int):
+            wanted_size = (self.output_size, self.output_size)
+        elif isinstance(self.output_size, tuple) or isinstance(self.output_size, list):
+            wanted_size = self.output_size
+        else:
+            raise AssertionError("Output size should be either a list, a tuple or an int")
```

Comment:
Let's move this to the constructor (it's strange to throw an error about constructor arg only when we perform inference)